src/java.base/share/classes/java/lang/Class.java

```diff
@@ -638,9 +640,7 @@ public boolean isValue() {
         if (!PreviewFeatures.isEnabled()) {
             return false;
         }
-         if (isPrimitive() || isArray() || isInterface())
-             return false;
-        return ((getModifiers() & Modifier.IDENTITY) == 0);
+        return !isIdentity();
```

Comment:
Copy the lines from PR#1499.
Nothing is value unless preview is enabled.

```suggestion
        return PreviewFeatures.isEnabled() ? !isIdentity() : false;
        
```

Reply:
This preview switch is retained right above the first deletion in this small piece of diff.